src/sqlancer/common/ast/newast/Select.java

```diff
@@ -44,5 +44,7 @@ public interface Select<J extends Join<E, T, C>, E extends Expression<C>, T exte
 
     Expression<C> getHavingClause();
 
+    Expression<C> toSum();
```

Comment:
Perhaps a comment would help here to understand the method and its purpose.

Reply:
i removed it after reviewing since it makes more sense for the generator to handle this